### PR TITLE
Added removing queried missed notification

### PR DESCRIPTION
**Pull request by ValentinTyhonov**

### CODE REVIEW

File: src/test/resources/MissedNotificationOutput.json [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+["{"subscriptionName": "Subscription_1","AggregatedObject": {"fileInformation": [{"extension": "jar","classifier": ""}],"buildCommand": null,"testCaseExecutions": [{"testCaseFinishEventId": "11109351-41e0-474a-bc1c-f6e81e58a1c9","testCaseStartedTime": 1481875925916,"testCaseStartedEventId": "cb9d64b0-a6e9-4419-8b5d-a650c27c59ca","testCaseFinishedTime": 1481875935919,"testCase": {"conclusion": "SUCCESSFUL","verdict": "PASSED","tracker": "My Other Test Management System","id": "TC5","uri": "https://other-tm.company.com/testCase/TC5"}}],"id": "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43","time": 1481875891763,"type": "ARTIFACT_1","gav": {"groupId": "com.mycompany.myproduct","artifactId": "sub-system","version": "1.1.0"}}}"]
```
Comment (tobiasake) [inline src/test/resources/MissedNotificationOutput.json]: What is this file used for? Is it output generated file from EI?

Comment (ValentinTyhonov) [reply src/test/resources/MissedNotificationOutput.json]: This is response of /queryMissedNotifications/{subscriptionName} call. It is used for test.

Comment (tobiasake) [reply src/test/resources/MissedNotificationOutput.json]: Ok,, so it is used for Expect result in test.

Comment (ValentinTyhonov) [reply src/test/resources/MissedNotificationOutput.json]: Yes

File: src/test/java/com/ericsson/ei/queryservice/test/QueryServiceTest.java [hunk 2]
```diff
@@ -149,19 +149,30 @@ public void processMissedNotificationTest() {
...
 
+    @Test
+    public void deleteMissedNotificationTest() {
+        Iterable<Document> responseDB = mongoClient.getDatabase(missedNotificationDataBaseName)
+            .getCollection(missedNotificationCollectionName).find();
+        Iterator itr = responseDB.iterator();
+        String response = itr.next().toString();
+        LOG.debug("The inserted doc is : " + response);
+        boolean removed = processMissedNotification.deleteMissedNotification("Subscription_1");
+        assertEquals(true, removed);
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/queryservice/test/QueryServiceTest.java]: I prefer that you double check with a query. The result from query should be empty.

**Pull request merged**